engineapi/instance_manager.go

```diff
@@ -315,19 +316,27 @@ func getDataEngineFromInstanceProcess(p *imapi.Instance) longhorn.DataEngineType
 	return longhorn.DataEngineType(p.BackendStoreDriver)
 }
 
-func getTypeForInstance(instanceType longhorn.InstanceType, portCount int32) longhorn.InstanceType {
+func getTypeForInstance(instanceType longhorn.InstanceType, name string) longhorn.InstanceType {
 	if instanceType != longhorn.InstanceType("") {
```

Comment:
Can we have a `longhorn.InstanceNone` to replace the empty string here?

Reply:
Fixed, thanks